hw_13/conftest.py

```diff
@@ -0,0 +1,69 @@
+import json
+import pytest
+from hw_13.CONSTANTS import ROOT_DIR
+from hw_13.page_objects.blog_page.blog_page import BlogPage
+from hw_13.page_objects.login_page.login_page import LoginLink
+from hw_13.utilities.configuration import Configuration
+from hw_13.utilities.driver_factory import DriverFactory
+
+
+def pytest_configure(config):
+    """
+    This function register an additional marker
+    :param config:
+    """
+    config.addinivalue_line(
+        "markers", 'regression: for regression'
+    )
+    config.addinivalue_line(
+        "markers", 'smoke: for smoke'
+    )
+
+
+@pytest.fixture(scope="session")
+def env():
+    """
+    This fixture gives access to configuraton.json
+    :return: dict
+    """
+    with open(f'{ROOT_DIR}/configurations/configuration.json') as f:
+        data = f.read()
+        json_to_dict = json.loads(data)
+    configs = Configuration(**json_to_dict)
+    return configs
+
+
+@pytest.fixture
+def create_driver(env):
+    """
+    This fixture creates friver
+    :param env: base_url
+    :return: driver
+    """
+    browser = env.browser_id
+    get_base_url = env.base_url
```

Comment:
you do not need a variable for this, it is used only one time. You can call env.browser_id directly in the function